### PR TITLE
Fix the broken `backButton` in the “Edit” screen

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/7103/bug-the-x-button-in-the-edit-screen-is-broken

### CODE REVIEW

**Comment by janhavisinghh** [comment]
@msasikanth this is a fix we need to hotfix in `DEMO` so can we please review this on priority?

File: CHANGELOG.md [hunk 1]
```diff
@@ -78,6 +78,7 @@
 - Add highlight when newly scanned bp passports are added
 - Index `patientUuid` in tables that are joined in `OverdueAppointment` database view
 - Migrate `RegistrationPinScreen` to use view effects
+- Fix broken back button in `EditPatientScreen`
```
Comment (msasikanth) [inline CHANGELOG.md:81]: Ideally this should be in the fixes section, since it's in demo and we are hotfixing it. But I am merging this in for now.

**Review by msasikanth — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (janhavisinghh) [reply CHANGELOG.md:81]: umm but this is not out there to the users yet so them seeing this as a fix would not make sense. @msasikanth

Comment (janhavisinghh) [reply CHANGELOG.md:81]: We caught this bug internally in a pre-production environment which is why I thought we can have this in the internal section

**Review by msasikanth — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:82]: Yeah, that's fair. So far, if it goes into demo and we have to fix an bug fix, we place it in `Fixes` section of the changelog.

**Pull request merged**